pkg/services/sqlstore/active_node.go

```diff
@@ -85,6 +91,10 @@ func InsertActiveNodeHeartbeat(cmd *m.SaveActiveNodeCommand) error {
 				sqlog.Debug(errmsg, "error", err)
 				return errors.New(errmsg + ": " + err.Error())
 			}
+			if cmd.ParticipantLimit > 0 && partID == int64(cmd.ParticipantLimit) {
```

Comment:
This condition looks incorrect to me. Because we want only one node processing cleanup. So partitionId for that node should be 0 but here it is checking for 'partId = 1' as partitionLimit=1.